src/main/java/wolox/training/config/ApplicationConfig.java

```diff
@@ -0,0 +1,14 @@
+package wolox.training.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import wolox.training.services.OpenLibraryService;
+
+@Configuration
+public class ApplicationConfig {
+
+    @Bean
+    public OpenLibraryService openLibraryService() {
```

Comment:
If you make OpenLibraryService a service bean, you don't have to do this. Remove this configuration